packages/suite/src/views/settings/device/index.tsx

```diff
@@ -166,22 +166,21 @@ const Settings = ({ device, locks, applySettings, changePin, openModal, goto }:
                     <TextColumn
                         title={<Translation id="TR_FIRMWARE_VERSION" />}
                         description={
-                            <Translation
-                                id="TR_YOUR_CURRENT_FIRMWARE"
-                                values={{ version: getFwVersion(device) }}
-                            />
+                            <>
+                                <Translation
+                                    id="TR_YOUR_CURRENT_FIRMWARE"
+                                    values={{ version: getFwVersion(device) }}
+                                />
+                                {device.isBitcoinOnly && ' (bitcoin-only)'}
```

Comment:
as we decided, do not put `isBitcoinOnly` to the Device.
Make suuite utility out of it